onnxruntime/core/external_ops/pyop.h

```diff
@@ -0,0 +1,239 @@
+#pragma once
+#ifdef _WIN32
+#include <Windows.h>
+#define LIB_PYOP           "onnxruntime_pyop.dll"
+#define LOAD_PYOP_LIB(n)   LoadLibraryA(n)
+#define LOAD_PYOP_SYM(h,n) GetProcAddress(h,n)
+#define UNLD_PYOP_LIB(h)   FreeLibrary(h)
+#else
+#ifdef __APPLE__
+#define LIB_PYOP           "./libonnxruntime_pyop.dylib"
+#else
+#define LIB_PYOP           "./libonnxruntime_pyop.so"
+#endif
+#define LOAD_PYOP_LIB(n)   dlopen(n,RTLD_NOW|RTLD_GLOBAL)
+#define LOAD_PYOP_SYM(h,n) dlsym(h,n)
+#define UNLD_PYOP_LIB(h)   dlclose(h)
+#define HMODULE            void*
+#include "dlfcn.h"
+#endif
+#include "core/framework/ml_value.h"
+#include "core/session/onnxruntime_cxx_api.h"
+#include "core/framework/op_kernel_context_internal.h"
+#include <iostream>
+#include <vector>
+#include <unordered_map>
+
+using ONNX_TYPES = std::vector<ONNXTensorElementDataType>;
```

Comment:
All uppercase is usually for macros, this is just a using? (comment applies to the other ones like this too)

Reply:
Good point.
Renamed.